.gitignore

```diff
@@ -0,0 +1,17 @@
+# build output
+build/
+
+# intermediate output
+**/Output/
+/test/**/*.yaml
```

Comment:
We need to keep the yaml within https://github.com/coredac/dataflow/tree/main/test/arch_spec.